### DISCUSSION TITLE
Sending ted and eForm of the same type of notice

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#694] [Q&A] [answered]
When a type F02 notice has been sent (in TED) and the notice is sent again but with eForms, should a subtype 16 be sent even if it had no changes, right?

Thanks in advance

**Comment by pdonohoe** [comment, marked as answer]
If a notice has been published in TED XML, it should not be sent again for publication in any format, unless there are changes.

**Reply by AlexAS81** [reply]
I mean publishing the file but now sending an eForm.

This case:

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/d7343a00-486a-40d1-bdd6-7fdf4b516ab3)

**Comment by YvesJo** [comment]
F02 (TEDXML) and Notice Subtype 16 (eForms) are both Contract Notices and there is no reason to have twice such a notice for a given procedure unless there is a change in the procedure or one of its components, in which case a change notice would be needed.
OPP-090 would be useful when Contract Award Notices would be published and the Contract Notice only exists in TEDXML.

**Reply by AlexAS81** [reply]
What you mean by this is that if an F02 has been published, a new 16 (eform) should not be generated if the processor decides to republish it for bidding, right?

**Reply by AlexAS81** [reply]
And if the processor has added any modification regarding F02, they will no longer be able to send an F14 beyond October 24. What should be done in those cases?

**Reply by YvesJo** [reply]
Any change to a TEDXML notice will have to be performed submitting an eForms change notice when submitting TEDXML notices will not be allowed anymore.
Therefore, once submission of TEDXML is forbidden, a change to the procedure defined by a F02 notice will need to be performed using the appropriate Notice Subtype (i.e. 16 in your scenario) with the Change section specified. This new eform notice will also have an associated Procedure ID and the link to the previous (TEDXML) notice will be done using BT-758. OPP-090 is not expected in this case.

**Reply by AlexAS81** [reply]
Therefore we would have to generate a type 16 with a change section?

**Reply by YvesJo** [reply]
if changes are needed, yes
